app/components/ParticipantInformation/DependentValidationInputField.js

```diff
@@ -41,6 +41,11 @@ class DependentValidationInputField extends Component {
     return !this.state.isValid ? 'red' : '';
   };
 
+  onFocusTextInput = () => {
```

Comment:
@limkimsan please consider creating a custom numeric field with onFocusTextInput() handling then reuse this custom component instead.